dept44-models/src/main/java/se/sundsvall/dept44/models/api/paging/AbstractParameterPagingBase.java

```diff
@@ -0,0 +1,55 @@
+package se.sundsvall.dept44.models.api.paging;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Min;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.domain.Sort;
+import se.sundsvall.dept44.models.api.paging.validation.MaxPagingLimit;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.Integer.parseInt;
+import static org.springframework.data.domain.Sort.DEFAULT_DIRECTION;
+
+/**
+ * Model class to extend when requesting paged result. Should be used as query parameters. *
+ * See {@link PagingMetaData} for response.
+ *
+ * Use property 'dept44.models.api.paging.max.limit' to change default max allowed page size.
+ */
+@EqualsAndHashCode
+@Getter
+@Setter
+public abstract class AbstractParameterPagingBase {
+
+	private static final String DEFAULT_PAGE = "1";
+	private static final String DEFAULT_LIMIT = "100";
+
+
+	@Schema(description = "Page number", example = "1")
```

Comment:
For extra clarity, maybe add something about the page numbering being 1-based to the schema description? Yes, the example states one, but still

Reply:
Minimum added